src/test/scala/net/ruippeixotog/scalascraper/browser/BrowserSpec.scala

```diff
@@ -103,8 +101,7 @@ class BrowserSpec extends Specification with BrowserHelper with TestServer {
       }
 
       "parse correctly HTML from a file" in {
```

Comment:
Can you duplicate this test and make one of them use `parseFile`, so we can test both methods here? Then we have "parse correctly HTML from a file" and "parse correctly HTML from a resource" tests.